Andela-Ray/PropertyPro-lite
# #167242646 Update UI templates

**What does this PR do?**
Update the UI templates based on LFA's feedback

**Description of Task to be completed?**
-Update CSS
-Update JS
-Update all views

**How should this be manually tested?**
After cloning the repository cd into it and run templates/index.html. You will observe the updated UI.

**What are the relevant pivotal tracker stories?**
#167242646
